src/app-layout/__integ__/app-layout-focus-delegation.test.ts

```diff
@@ -6,17 +6,32 @@ import useBrowser from '@cloudscape-design/browser-test-tools/use-browser';
 import createWrapper from '../../../lib/components/test-utils/selectors';
 import { viewports } from './constants';
 
+class FocusDelegationPage extends BasePageObject {
+  getActiveElement() {
+    return this.browser.execute(() => {
+      return document.activeElement;
+    });
+  }
```

Comment:
Looks like this addition is unused

Reply:
I was using it for debugging. Thanks for pointing this out